UltraStar Play/Assets/Common/Graphics/ImageManager.cs

```diff
@@ -84,6 +82,8 @@ private static void RemoveUnusedSpritesFromCache()
                 RemoveCachedSprite(cachedSprite);
             }
         }
+        // Trigger garbage collection of used resources.
+        Resources.UnloadUnusedAssets();
```

Comment:
why does UnloadUnusedAssets get called here? It should not be necessary to call this at all and according to a paid-for fulltime Unity games dev that I just asked, this should not be called at all. Use GemoObject.Destory() (as is already done below in the code) to have the unity code clean up the object and also free up the GPU memory.

Reply:
I checked it with the profiler and found that you are right. It is not needed. Texture count is still going down when scrolling songs.